diff --git a/node_modules/express-graceful-exit/index.d.ts b/node_modules/express-graceful-exit/index.d.ts
index 551de8f..343c209 100644
--- a/node_modules/express-graceful-exit/index.d.ts
+++ b/node_modules/express-graceful-exit/index.d.ts
@@ -1,7 +1,7 @@
 import { Server } from 'net'
 import { Express, NextFunction, Request, RequestHandler, Response } from 'express'
 
-namespace GracefulExit {
+declare namespace GracefulExit {
   interface Configuration {
     errorDuringExit?: boolean
     performLastRequest?: boolean
